README.md

```diff
@@ -127,16 +147,23 @@ Edit the following code snippet. Add the `<Router>` provider component, and rewr
 class App extends Component {
   render () {
     return (
+      <Router>
       <div>
         <h1>Welcome to My shopping site</h1>
         <nav>
           <a href="/">Home</a>
           <a href="/products">Products</a>
           <a href="/cart">Shopping Cart</a>
         </nav>
+        <main>
+        <Switch>
+        <Route path="/products" component={Products} />
+        <Route path="/cart" component={Cart} />
+        <Route path="/" component={Home} />
```

Comment:
Closing tag `</Switch>` was missing but the rest of the implementation here is solid